packages/@adobe/spectrum-css-temp/components/slider/skin.css

```diff
@@ -327,3 +329,46 @@ governing permissions and limitations under the License.
     margin-inline-end: var(--spectrum-slider-label-gap-x);
   }
 }
+@media (forced-colors: active) {
```

Comment:
Current:
![slider](https://user-images.githubusercontent.com/9661127/175711613-5003eb64-747d-46fd-9542-98bdf78fd1cb.PNG)


XD:
<img width="218" alt="image" src="https://user-images.githubusercontent.com/9661127/175713505-7700adce-2478-4836-ba4c-a075e645e186.png">


I think I'm fine with the track not changing colors since it distinguishes it from isFilled/fillOffset, but is it possible to get that black dot in the middle of the pressed slider handle? Pretty minor IMO so if it hard to do I'm fine with the current experience